operator/build/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM quay.io/operator-framework/ansible-operator:v0.11.0
+#FROM quay.io/mew2057/ansible-operator:dev
```

Comment:
If we don't need these two changes in this file, could remove.  Probably was useful to switch during development

Reply:
Removed